x-pack/plugins/canvas/__fixtures__/function_specs.ts

```diff
@@ -9,6 +9,5 @@ import { functions as browserFns } from '../canvas_plugin_src/functions/browser'
 import { ExpressionFunction } from '../../../../src/plugins/expressions';
 import { initFunctions } from '../public/functions';
 
-export const functionSpecs = browserFns
-  .concat(...(initFunctions({} as any) as any))
-  .map((fn) => new ExpressionFunction(fn()));
+export const getFunctionSpecs = () =>
```

Comment:
Is this change necessary still, or is it a remnant of when you were doing async function definitions?

Reply:
Yes, you are right, that is a remnant of the `async` changes.

Reply:
Done.